pages/base_page.py

```diff
@@ -16,3 +16,59 @@ def is_element_present(self, how, what):
         except NoSuchElementException:
             return False
         return True
+
+    def should_be_exact_text(self, expected_text, real_text):
+        assert expected_text in real_text, f"Expected text: {expected_text}, but got: {real_text}"
```

Comment:
```suggestion
        assert expected_text in real_text, f"Should be text '{expected_text}' in '{real_text}'"
```
Давай сделаем чуть более информативно.